app/src/docs/v1.api-spec.yaml

```diff
@@ -231,35 +362,77 @@ components:
             - fizz@gov.bc.ca
         context:
           type: object
-          description: A freeform JSON object of key-value pairs.  All keys must be alphanumeric or underscore.
+          description: >-
+            A freeform JSON object of key-value pairs.  All keys must be
+            alphanumeric or underscore.
           example:
             something:
               greeting: Hello
               target: World
             someone: user
-    EmailRequest:
+        delay:
+          type: integer
+          description: number of milliseconds to delay this message before queueing
+          example: 60000
+        to:
+          type: array
+          items:
+            type: string
+          description: >-
+            An array of recipients email addresses that will appear on the To:
+            field
+          example:
+            - baz@gov.bc.ca
+    EndpointStatus:
+      type: object
+      required:
+        - name
+        - endpoint
+        - healthCheck
+        - authenticated
+        - authorized
+      properties:
+        name:
+          type: string
+          description: The name of the endpoint
+          example: Example API
+        endpoint:
+          type: string
+          description: Base endpoint URL
+          example: 'https://example.com/v1/'
+        healthCheck:
+          type: boolean
+          description: Is the endpoint reachable
+        authenticated:
```

Comment:
not sure if we need authenticated and authorized - they get a 401 on /health if they don't have authorization.  let's trim out what we don't need.

Reply:
Think we should trim this out? We just happen to not need authentication/authorization with apps.smtp, but that isn't to say that other SMTP endpoints will be that permissive as well. Granted it is redundant in our case, but this endpoint is meant to be generic in format so that if there was an aggregator health checker in the future we'd have an endpoint that provides that information in a relatively generic way.

Reply:
We have to think what is this endpoint for?  are we using healthCheck to return a status to all our endpoints? /email, /emailMerge etc?  Is the returning a catalog of endpoints and whether you have access? If it is just in place so we can verify we are up and responding, then keep only what we need.  If this is in place to check the smtp server and our access to it, let's discuss on how to do that.  I'd say it's true/false, with potentially a code as to why it's false (ex. our credentials to the server failed - 401).

Reply:
I was under the impression we were fitting in the latter case - make sure the API has the ability to access the things it needs to and report on whether that is true or not.

Reply:
Yeah the heathcheck is returning relying component status. Maybe _endpoint_ isn't the clearest term, but that's what we went with for other implementations. Can rethink later on,

Reply:
I'm marking the /health path as (DRAFT) for now so we remember it hasn't been ironed down yet